src/main/java/com/lothrazar/cyclic/block/solidifier/TileSolidifier.java

```diff
@@ -173,13 +185,48 @@ private void findMatchingRecipe() {
       return;
     }
     currentRecipe = null;
+    if (!inputSlots.getStackInSlot(0).isEmpty()) {
```

Comment:
This distributes the items between the slots where possible for helping automation.